environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
  - python>=3.7
  - pyrosetta
  - numpy=1.16.3
+ - numba
  - matplotlib
  - pandas=0.24.2
```

Comment:
can you add this in here? then I don't need to open a commit for it...
` - pyyaml`